notebooks/06-mnist-tpu-training.ipynb

```diff
@@ -80,7 +80,7 @@
         "id": "AYGWh10lRaF1"
       },
       "source": [
-        "! pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.7-cp36-cp36m-linux_x86_64.whl"
+        "! pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.7-cp37-cp37m-linux_x86_64.whl"
```

Comment:
any documentation we should add to handle future version changes?

Reply:
Just follow Google announcement on Colab upgrades....
Unfortunately there is no simple way to test colab in CI

Reply:
In the docs we show these commands here:
https://pytorch-lightning.readthedocs.io/en/latest/advanced/tpu.html#colab-tpus

maybe this is more future proof, and we should include them in the notebooks?
I have totally no clue here, found this fix randomly and only did this because on call tbh.

Reply:
https://twitter.com/GoogleColab/status/1365062244225060865?s=20